electron/main/index.ts

```diff
@@ -5,12 +5,12 @@ process.env.PUBLIC = app.isPackaged
   : join(process.env.DIST_ELECTRON, '../public');
 
 /* eslint-disable @typescript-eslint/unbound-method */
-import { Atom, Tatom } from '@bitgo-beta/sdk-coin-atom';
-import { Bld, Tbld } from '@bitgo-beta/sdk-coin-bld';
-import { Hash, Thash } from '@bitgo-beta/sdk-coin-hash';
-import { Injective, Tinjective } from '@bitgo-beta/sdk-coin-injective';
-import { Sei, Tsei } from '@bitgo-beta/sdk-coin-sei';
-import { Tia, Ttia } from '@bitgo-beta/sdk-coin-tia';
```

Comment:
Hw about osmo and zeta ?

Reply:
good catch, curious I don't have it in - locally it seems to be updated and there is nothing to commit/push

Reply:
^@DinshawKothari ignore above comment. So for both osmo/zeta I used the alpha versions when I first made the change, so that's why you don't see it here.